kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -326,7 +326,7 @@ import ShowFor from "sumo/js/showfor";
           // If form isn't valid, click the modal submit button
           // so the validation error is shown. (I couldn't find a
           // better way to trigger this.)
-          $modal.find('button[type="submit"]').click();
+          $modal.find('button[type="submit"]').on("click", );
```

Comment:
This should be a trigger.
```suggestion
          $modal.find('button[type="submit"]').trigger("click");
```